src/services/command/command.ts

```diff
@@ -13,7 +13,7 @@ import type { Context } from "./context.js";
  * 1. Every command corresponds to a file inside of src/commands/
  * 2. The order determines the order of commands in the README
  */
-export const Commands = ["dev", "deploy", "status", "push", "pull", "open", "list", "login", "logout", "whoami", "version"] as const;
+export const Commands = ["dev", "deploy", "status", "push", "pull", "open", "list", "login", "logout", "whoami", "version", "add"] as const;
```

Comment:
Should we add `add` a little higher up in the README?

Maybe between `pull` and `open`?